.travis.yml

```diff
@@ -46,11 +46,13 @@ addons:
     sources:
       - elasticsearch-5.x
     packages:
+      - cmake
       - swig
       - elasticsearch
       - gettext
       - librsvg2-bin
       - pngcrush
+      - uuid
```

Comment:
We are seeing `uuid does not exist` errors in travis tests because of how our makefile is structured and uses `uuid` so I added it. Unrelated to this PR though